lib/jekyll-seo-tag/json_ld.rb

```diff
@@ -40,7 +40,7 @@ def json_author
         return unless author
         {
           "@type" => "Person",
-          "name"  => author["name"],
+          "name"  => author["name"] || page["author"],
```

Comment:
Theoretically, the `author` object should resolve to the actual author (be it stored as hash or a string, in the post, in data, or in config), and consistently return a hash, so we don't have to do this.